experimental/generation/generator/packages/library/templates/standard/en-us/percentageEntity.en-us.lu.lg

```diff
@@ -1,9 +1,11 @@
 [standard.en-us.lg](standard.en-us.lg)
 
 # filename
-- ${builtinEntityFilename('percentage')}
+- ${builtinEntityFilename('number')}
 
 # template
 - ```
-@ prebuilt percentage
+@ prebuilt number
```

Comment:
>number [](start = 11, length = 6)

This is incorrect.  We should use the percentage prebuilt.  The resolution is a number and the json schema type should be number.